Library/Homebrew/test/helper/fixture.rb

```diff
@@ -0,0 +1,13 @@
+module Fixture
+  module_function
+
+  FIXTURE_DIR = File.expand_path(File.join(__FILE__, "..", "..", "fixtures")).freeze
+
+  def path(name)
```

Comment:
`HOMEBREW_TEST_FIXTURES` as a constant should be sufficient and can be joined with `/` rather than needing a new method and class defined.